### PR TITLE
DF/DA-594(Localization)

### PR DESCRIPTION
**Pull request by dacianfl**
For this ticket I've used an intelij plugin called Flutter i18n. This plugin works with arb files which are located inside res/values directory and this file type has the same structure as a json file.
This plugin takes all the arb files and generates a class called S full of static fields with every key from arb file.

### CODE REVIEW

File: marty_flutter/web/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (vladms) [inline marty_flutter/web/index.html]: @dacianfl Please check if you need this HTML

File: marty_flutter/lib/screens/navigation/menu_item_list.dart [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import 'package:configurable_expansion_tile/configurable_expansion_tile.dart';
 import 'package:flutter/material.dart';
+import 'package:marty_flutter/generated/i18n.dart';
 import 'package:marty_flutter/models/ScreenType.dart';
 import 'package:marty_flutter/themes/app_colors.dart';
 import 'package:marty_flutter/themes/dimens.dart';
-import 'package:marty_flutter/themes/strings.dart';
 
 class MenuItemList extends StatelessWidget {
```
Comment (vladms) [inline marty_flutter/lib/screens/navigation/menu_item_list.dart]: @dacianfl Please format the code for the whole file(CMD + ALT + L)

File: marty_flutter/web/index.html (hunk 1 above)
Comment (dacianfl) [reply marty_flutter/web/index.html]: @vladms  I have flutter web enabled, so when I've ran 'flutter start' command it added this file too. I suppose flutter needs it the same as it needs android and ios packages.

**Comment by vladms** [comment]
@dacianfl  Please merge with dm/nav_menu

**Comment by dacianfl** [comment]
> @dacianfl Please merge with dm/nav_menu

**Pull request merged**